server/routes/deleteObject.js

```diff
@@ -8,16 +8,17 @@ app.use(cors());
 app.use(bodyParser.json());
 import { minioClient } from '../minioConfig.js';
 import { get_user_bucket, delete_file } from '../Database_queries/queries.js';
+import { logger, log } from '../logger.js';
```

Comment:
The logger import should be consistent. If you are using `log` for logging, consider removing the `logger` import if it's not used. This will help in keeping the code clean.
*This comment was auto-generated by **Code Bot** to provide suggestions and improvements.*